### PR TITLE
feat: format unused-path at Page and Redirects

### PR DESCRIPTION
**Pull request by igorbrasileiro**

### CODE REVIEW

**Review by lucis — APPROVED** [review]

**Comment by lucis** [comment]
Question: Shouldn't we make it a bit clearer that this means "path already present on the site", and not a random path?

Not sure if we're locking anything here

**Comment by igorbrasileiro** [comment]
> Question: Shouldn't we make it a bit clearer that this means "path already present on the site", and not a random path?

I didn't get it what you mean, can you rephrase? As far as I understood what you said, the admin that handles this `format path-uri`
